### DISCUSSION TITLE
MoneyExtension

### DISCUSSION THREAD

**Discussion by JotaPeCarvalho** [balta-io/forum#227] [Tira Dúvidas] [answered]
Olá, tudo bem? Não achei o repositório do código money extension, podem me ajudar?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @JotaPeCarvalho, como vai?
O repositório segue sempre o código do curso:
https://github.com/balta-io/<<CÓDIGO DO CURSO>>

Por exemplo:
https://github.com/balta-io/7189

Além disso, abaixo de toda aulas (Vídeo) tem um botão que te leva direto ao repositório:
![image](https://user-images.githubusercontent.com/965305/212895339-7ea1bfed-8e52-4249-83b0-3dd2a607363d.png)

**Reply by JotaPeCarvalho** [reply]
Olá, então, estou atrás do código do projeto "Money Extension", tanto o botão quanto o código do curso levam a outro projeto, poderia me ajudar?

**Reply by andrebaltieri** [reply]
@JotaPeCarvalho veja se é este: https://github.com/andrebaltieri/money-extensions
